### ISSUE TITLE
Running AWS Lambda locally fails with "module initialization error". Debug local, same app runs fine.

### ISSUE THREAD

**Issue by stevewoolley** [aws/aws-toolkit-jetbrains#753]
**Describe the bug**
Using HelloWorld template to build SAM app.
Running locally HelloWorldFunction throws:
> {"errorMessage": "module initialization error"}
module initialization error: 'module' object has no attribute 'Queue'

Debug locally runs fine:
> {"body": "{\"message\": \"hello world\", \"location\": \"4.7.137.131\"}", "statusCode": 200}

**To reproduce**
1. Use AWS Serverless Application plugin to create project
1. create virtualenv using 2.7 and 3.6
1. Use SAM template "AWS SAM Hello World"
1. Add aws-sam-cli to project interpreter
1. Set AWS toolkit settings to "show gutter icons for all potential AWS Lambda handlers" 
1. Select lambda icon in sidebar of app.py and "run Local"
1. Edit configuration dialog appears. Set template as template.yaml. Set input to event template for "Hello World"
1. Run 

Output:

> /Users/swoolley/workspace/snerted/venv/bin/sam local invoke --template "/private/var/folders/3c/cbwj1xf169d6pdffr5dgzl_00000gn/T/[Local] app.lambda_handler-template2.yaml" --event "/private/var/folders/3c/cbwj1xf169d6pdffr5dgzl_00000gn/T/[Local] app.lambda_handler-event2.json" HelloWorldFunction
2019-01-17 11:36:39 Found credentials in environment variables.
2019-01-17 11:36:39 Invoking app.lambda_handler (python2.7)
2019-01-17 11:36:39 Decompressing /var/folders/3c/cbwj1xf169d6pdffr5dgzl_00000gn/T/6976455448378379209.zip
Fetching lambci/lambda:python2.7 Docker container image......
2019-01-17 11:36:42 Mounting /private/var/folders/3c/cbwj1xf169d6pdffr5dgzl_00000gn/T/tmpt1mWyu as /var/task:ro inside runtime container
START RequestId: 9b3f5a1d-b01a-4e06-aa64-87b4d640c484 Version: $LATEST
{"errorMessage": "module initialization error"}
module initialization error: 'module' object has no attribute 'Queue'
END RequestId: 9b3f5a1d-b01a-4e06-aa64-87b4d640c484
REPORT RequestId: 9b3f5a1d-b01a-4e06-aa64-87b4d640c484 Duration: 230 ms Billed Duration: 300 ms Memory Size: 128 MB Max Memory Used: 16 MB

**Expected behavior**
> /Users/swoolley/workspace/snerted/venv/bin/sam local invoke --template "/private/var/folders/3c/cbwj1xf169d6pdffr5dgzl_00000gn/T/[Local] app.lambda_handler-template1.yaml" --event "/private/var/folders/3c/cbwj1xf169d6pdffr5dgzl_00000gn/T/[Local] app.lambda_handler-event1.json" HelloWorldFunction --debug-port 52924 --debugger-path /Applications/PyCharm.app/Contents/helpers/pydev --debug-args "-u /tmp/lambci_debug_files/pydevd.py --multiprocess --port 52924 --file"
2019-01-17 11:36:09 Found credentials in environment variables.
2019-01-17 11:36:09 Invoking app.lambda_handler (python2.7)
2019-01-17 11:36:09 Decompressing /var/folders/3c/cbwj1xf169d6pdffr5dgzl_00000gn/T/5068592744554213578.zip
Fetching lambci/lambda:python2.7 Docker container image......
2019-01-17 11:36:12 Mounting /private/var/folders/3c/cbwj1xf169d6pdffr5dgzl_00000gn/T/tmpikly0M as /var/task:ro inside runtime container
Connected to pydev debugger (build 183.5153.39)
START RequestId: 68ca9509-fb56-4e42-873b-7270db203fea Version: $LATEST
END RequestId: 68ca9509-fb56-4e42-873b-7270db203fea
REPORT RequestId: 68ca9509-fb56-4e42-873b-7270db203fea Duration: 1100 ms Billed Duration: 1200 ms Memory Size: 128 MB Max Memory Used: 25 MB
{"body": "{\"message\": \"hello world\", \"location\": \"4.7.137.131\"}", "statusCode": 200}

**Your Environment**
 - OS: MacOS 10.14.2
 - JetBrains' Product: PyCharm
 - JetBrains' Product Version: 2018.3
 - Toolkit Version: 1.1 and 1.2-SNAPSHOT
 - SAM CLI Version: 0.10.0
 - JVM/Python Version: Python 2.7 and 3.6

**Additional context**
Docker on my mac was recently upgraded to 2.0.0.2
Interesting that I had to add /Applications to File Sharing settings on Docker Desktop. Debug was throwing different error until I did this.

**Comment by rli** [comment]
Looks like dependencies being pulled in by `aws-sam-cli` are clobbering each other. You should either install the SAM CLI into its own virtual environment, or switch to the Homebrew install method.

**Comment by stevewoolley** [comment]
My first reaction to your response is was I was already using SAM CLI in virtualenv. However, I blew away all my python interpreters (both brew and pyenv). Then reinstalled.
The I happened to look at the settings for AWS under Preferences/Tools/AWS and noticed that the 'Global AWS Toolkit Settings' had the SAM CLI executable referenced as living under a separate projects virtualenv. I then reinstalled:

> brew tap aws/tap
brew install aws-sam-cli

then manually set the SAM CLI executable location to:
> /usr/local/bin/sam

and it now works.   Thanks for the response!
